packages/schemas/tables/aggregated_daily_active_users.sql

```diff
@@ -0,0 +1,15 @@
+create table aggregated_daily_active_users (
```

Comment:
Need some comments here for future reference.

Reply:
Updated.